esridump/esri2geojson.py

```diff
@@ -1,18 +1,33 @@
 from itertools import tee
+from functools import partial
+from shapely.geometry import shape, mapping
+from shapely.ops import transform
+import pyproj
 
-def esri2geojson(esrijson_feature):
+def esri2geojson(esrijson_feature, srid = 'epsg:4326'):
```

Comment:
Python style is usually no whitespace around `=`'s for arg defaults.

```suggestion
def esri2geojson(esrijson_feature, srid='epsg:4326'):
```